### PR TITLE
Physical_oM: Add MeshElement

### LINKED ISSUE #1451: Physical_oM: Add MeshElement class
#### Description:

From discussions with @al-fisher @pawelbaran @alelom 
Good to add a MeshElement class as a new physical element.
The new element will have a single property - a geometrical Mesh.
Usefull to be able to store a quite direct representation of BIM elements from Revit/IFC/Similar.
Parameters from the object to be stored in fragements corresponding to the particular application.

### PR DESCRIPTION
**Pull request by IsakNaslundBh**
### Issues addressed by this PR

Closes #1451

Adding a simple `MeshElement` class. First use case to be able to make a simplified representation of BIM objects, where parameters are to be stored in software specific fragments, and the Mesh property is to host the mesh representation of the object.

### Test files

To be tested in anger in various workflows in other toolkits. For this first push just good to get the object in for which code review should suffice.

### CODE REVIEW

**Comment by IsakNaslundBh** [comment]
@BHoMBot check compliance

**Comment by IsakNaslundBh** [comment]
@BHoMBot check required

File: Physical_oM/Elements/MeshElement.cs [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+using BH.oM.Base;
+using BH.oM.Geometry;
+using BH.oM.Quantities.Attributes;
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+
+namespace BH.oM.Physical.Elements
+{
+    [Description("Element defined by a geometrical mesh defining the 3-dimensional geometry of the element.\n" +
```
Comment (pawelbaran) [inline Physical_oM/Elements/MeshElement.cs:32]: Following the offline discussion, I would consider tweaking the description a slight bit
```suggestion
    [Description("Object representing a physical element, defined by a geometrical mesh that represents that element in 3-dimensional space.\n" +
```

Comment (al-fisher) [reply Physical_oM/Elements/MeshElement.cs:32]: Yes - I like this

Comment (IsakNaslundBh) [reply Physical_oM/Elements/MeshElement.cs]: Happy with this as well.
Hve commited your suggestion @pawelbaran :). Thanks!

**Comment by IsakNaslundBh** [comment]
@BHoMBot check compliance

**Comment by IsakNaslundBh** [comment]
@BHoMBot check required

**Comment by IsakNaslundBh** [comment]
@BHoMBot check versioning

**Comment by pawelbaran** [comment]
@BHoMBot check ready-to-merge

**Comment by IsakNaslundBh** [comment]
> Happy with the current version of the code, approving!

Thanks @pawelbaran !

**Pull request merged**